docs/en/01_Tutorials/02_Extending_A_Basic_Site.md

```diff
@@ -462,16 +462,19 @@ Nothing here should be new. The *StaffPage* page type is more interesting though
 	:::php
 	<?php
 	class StaffPage extends Page {
-		private static $db = array(
-		);
+
 		private static $has_one = array(
-			'Photo' => 'Image'
+			'Photo' => 'File'
```

Comment:
This is the hard part of this change. I don't have a better solution at the moment.

I think this needs to be addressed at the DBFile level before on the actual DataObject. Read on and get back to me. :D